qiskit_dynamics/solvers/solver_classes.py

```diff
@@ -224,97 +302,151 @@ def solve(
 
         Additional Information:
 
-            The behaviour of this method is impacted by the input type of ``y0``:
+        The behaviour of this method is impacted by the input type of ``y0``:
```

Comment:
General thought not for this specific PR: should there be more detailed documentation on how solvers/rotating frames etc work in the `solvers.__init__` documentation?

Reply:
yeah I agree and was thinking about this as well while making this PR. All of this is actually explained in the `models.__init__` documentation, but given that `Solver` has become the main interface it definitely makes sense to include some version of it there (or even transfer it from the `models` section). I'll make an issue for this.